pkgs/dart_model/benchmark/json_buffer.dart

```diff
@@ -25,7 +93,75 @@ class JsonBufferSerializeBenchmark extends BenchmarkBase {
 
   @override
   void run() {
-    serialized = JsonBuffer(LazyMap(mapKeys, (key) => key)).serialize();
+    serialized = JsonBuffer(LazyMap(mapKeys, (key) {
```

Comment:
Using a single canonical `mapKeys` list like this is cheating some, I don't think we will ever be able to use pre-existing lists for keys in practice because we are always providing a filtered view on objects (except maybe if somebody asks for _every_ field we could special case that).

Unless we always serialize null values too, but if we were going to do that I would probably advocate for an entirely different approach to serialization :).

Reply:
Yes, will see if this can be a bit more realistic.